CHANGEDB.php

```diff
@@ -794,9 +794,12 @@
 UPDATE `gibbonAction` SET category='Learn' WHERE gibbonModuleID=(SELECT gibbonModuleID FROM gibbonModule WHERE name='School Admin') AND name='Individual Needs Settings';end
 ALTER TABLE `gibbonPerson` CHANGE `departureReason` `departureReason` VARCHAR(100) NOT NULL;end
 INSERT INTO `gibbonNotificationEvent` (`event`, `moduleName`, `actionName`, `type`, `scopes`, `active`) VALUES ('First Aid Qualification Expiry', 'Staff', 'Manage Staff', 'Core', 'All', 'Y');end
+<<<<<<< HEAD
```

Comment:
These merge indicators appear to have made their way into the commit. They're generally added when resolving merge conflicts, and need removed before committing/PRing. Thanks!
```suggestion
```